governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Ratification Poll for Oracles Core Unit (ORA-001) Subproposal Set - June 14, 2021
+summary: Signal your support or opposition for these proposals being accepted and implemented in the Maker Protocol. 
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp12-adding-collateral-onboarding-core-unit/8037
```

Comment:
wrong link here


correct one: https://forum.makerdao.com/tag/ora-001

Reply:
We're using the mandate for the time being